index.php

```diff
@@ -68,7 +66,9 @@ function updateDropdown(District, City, Street) {
   } 
   
   if (this.id == 'Street') {
-    console.log('The Street is: ' + $(this).val());
+    var District = $('#District').val();
+    var City = $('#City').val();
+    updateDropdown(District, City, null);

```

Comment:
I'm passing city as null by default as we won't have that value anyway, but city has value, so this should work and you should see the dropdown filled with values.

You already had it almost done, only thing I done was just to call the right function in right place.